es6-classes/square.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super(area, perimeter);
```

Comment:
`area` and `perimeter` haven't been defined. You should be passing in a number here, utilizing your `width` parameter.